docker-compose.yml

```diff
@@ -70,7 +70,18 @@ services:
       mindfuel_activity:
         ipv4_address: 10.10.20.1
 
+  mongodb_contaier:
```

Comment:
Let's remove the Elastic Search container from docker-compose.yml altogether (line 53-73).